proofpoint-threat-response/info.json

```diff
@@ -60,10 +60,10 @@
   "operations": [
     {
       "operation": "get_list",
-      "title": "Get List",
+      "title": "Get List of Indicators",
```

Comment:
The action name should go by Get Indicators List, This is the standard which we follow for all get actions make the corresponding changes in all such get actions.